src/runtime/GlobalObjectBuiltinString.cpp

```diff
@@ -907,21 +907,71 @@ static Value builtinStringToLocaleUpperCase(ExecutionState& state, Value thisVal
     return builtinStringToUpperCase(state, thisValue, argc, argv, isNewExpression);
 }
 
-static Value builtinStringTrim(ExecutionState& state, Value thisValue, size_t argc, Value* argv, bool isNewExpression)
+enum BuiltinStringWhere : unsigned {
+    START,
```

Comment:
How about naming this enum as `StringTrimWhere` to keep the consistency with other enum definitions?

Reply:
I was not sure about the naming, this is a good idea! Changed it in the latest push.